### PR TITLE
YLP-2183 Monitoring alerts columns

### PR DESCRIPTION
**Pull request by celia-rouis**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/yourloops/test/integration/pages/home-page/home.hcp.spec.tsx [hunk 1]
```diff
@@ -254,7 +260,7 @@ describe('HCP home page', () => {
       dataNotTransferredFilterToggle: true,
       unreadMessagesFilterToggle: true
     })
-    checkDataGridAfterSinglePatientFilter(dataGridRow, 'Flag patient unread-messages@patient.frUnread Messages PatientDBLG110%20%30%N/AThe patient has sent you new messages')
+    checkDataGridAfterSinglePatientFilter(dataGridRow, 'Flag patient unread-messages@patient.frUnread Messages Patient.time-spent-out-of-range-icon-no-fill { fill: none }DBLG1N/AThe patient has sent you new messages')
```
Comment (TimCanu) [inline packages/yourloops/test/integration/pages/home-page/home.hcp.spec.tsx:263]: argh

File: packages/yourloops/components/patient-list/custom-cells.tsx [hunk 2]
```diff
@@ -101,42 +107,64 @@ export const PendingIconCell: FunctionComponent = () => {
   )
 }
 
-export const MonitoringAlertPercentageCell: FunctionComponent<MonitoringAlertPercentageCellProps> = ({ isMonitoringAlertActive, value }) => {
-  const { user } = useAuth()
+export const MonitoringAlertsCell: FunctionComponent<MonitoringAlertsCellProps> = ({ monitoringAlerts }) => {
+  const { t } = useTranslation()
   const theme = useTheme()
 
+  const isTimeSpentAwayFromTargetAlertActive = getMonitoringAlertActiveValueByType(monitoringAlerts, MonitoringAlertType.TimeSpentAwayFromTarget)
```
[the remark this answers is not in the document]
Comment (celia-rouis) [reply packages/yourloops/components/patient-list/custom-cells.tsx:114]: I guess I thought it was going to be much more complicated at first, I had even tried to anticipate the sorting... But you're 100% right, the method makes no sense right now :broom:

File: packages/yourloops/components/patient-list/models/enums/monitoring-alert-type.enum.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+export enum MonitoringAlertType {
```
Comment (TimCanu) [inline packages/yourloops/components/patient-list/models/enums/monitoring-alert-type.enum.ts:28]: I think that you no longer need this

**Review by TimCanu — APPROVED** [review]

**Pull request merged**